etc/PackageConfigurations/eos_package_description.json

```diff
@@ -1,66 +1,47 @@
-        {"src": "LICENSE.md",                       "dest": "" },
-        {"src": "README.md",                        "dest": "" },
-        {"src": "etc/PackageTemplate/package.json", "dest": "" },
-        {"src": "etc/PackageTemplate/Editor.meta",  "dest": "" },
-        {"src": "etc/PackageTemplate/Runtime.meta", "dest": "" },
+        {"src": "*.md",                             "dest": "/" },
+        {"src": "etc/PackageTemplate/package.json", "dest": ""  },
+        {"src": "etc/PackageTemplate/Editor.meta",  "dest": ""  },
+        {"src": "etc/PackageTemplate/Runtime.meta", "dest": ""  },
 
         {"src": "etc/PlatformSpecificAssets/EOS/*", "dest": "PlatformSpecificAssets~/EOS/", "recursive": true },
         {"src": "docs/*",                           "dest": "Documentation~/",              "recursive": true },
         {"src": "tools/bin/*",                      "dest": "bin~/",                        "recursive": true },
 
-        {"src": "Assets/EOS/*.png", "dest": "Runtime/EOS/"},
+        {"src": "Assets/EOS.meta",  "dest": "Runtime/"     },
+        {"src": "Assets/EOS/*.png", "dest": "Runtime/EOS/" },
                              
-        {"src": "Assets/Plugins/*",                                   "dest": "Runtime/"                         },
-        {"src": "Assets/Plugins/Source/Core/*",                       "dest": "Runtime/Core/"                    },
-        {"src": "Assets/Plugins/Source/Core/Utility/*.cs",            "dest": "Runtime/Core/Utility/"            },
-        {"src": "Assets/Plugins/Source/Core/Utility/Extensions/*.cs", "dest": "Runtime/Core/Utility/Extensions/" },
-        {"src": "Assets/Plugins/Source/Editor/*",                     "dest": "Editor/"                          },
-
-        {"src": "Assets/Plugins/Windows/*",                           "dest": "Runtime/Windows/"      },
-        {"src": "Assets/Plugins/Windows/x64/*",                       "dest": "Runtime/Windows/x64/"  },
-        {"src": "Assets/Plugins/Windows/x86/*",                       "dest": "Runtime/Windows/x86/"  },
-        {"src": "Assets/Plugins/Windows/Core/*",                      "dest": "Runtime/Windows/Core/" },
-        {"src": "Assets/Plugins/Windows/Editor/*",                    "dest": "Editor/Windows/"       },
+        {"src": "Assets/Plugins/*",                                "dest": "Runtime/"                             },
+        {"src": "Assets/Plugins/Source/Core.meta",                 "dest": "Runtime/"                             },
+        {"src": "Assets/Plugins/Source/Core/*",                    "dest": "Runtime/Core/"                        },
+        {"src": "Assets/Plugins/Source/Core/Utility/*",            "dest": "Runtime/Core/Utility/"                },
+        {"src": "Assets/Plugins/Source/Core/Utility/Extensions/*", "dest": "Runtime/Core/Utility/Extensions/"     },
+        {"src": "Assets/Plugins/Source/Editor/*",                  "dest": "Editor/",          "recursive": true  },
```

Comment:
It feels a bit off to include this recursively, since platform folders are added in, each platform should be added explicitly

Reply:
I agree

Reply:
This was fixed according to recommendations. Marking conversation as resolved.